framework/src/modules/auth/endpoint.ts

```diff
@@ -83,11 +82,11 @@ export class AuthEndpoint extends BaseEndpoint {
 
 		const accountAddress = cryptoAddress.getAddressFromPublicKey(transaction.senderPublicKey);
 
-		const store = this.stores.get(AuthAccountStore);
-		const account = await store.get(context, accountAddress);
+		const authAccountStore = this.stores.get(AuthAccountStore);
```

Comment:
Just checking, I see you changed the `store` var to `authAccountStore`,  any reason the same var on lines 49 & 110 did not get the update? Not a show stopper, just curious 🙂

Reply:
The reason is because I didn't touch those methods in this PR, but you're right. We should replace all the generic names with more specific ones 🙂 

Updated.